src/components/shared/SharedList.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Box from "@mui/material/Box";
+import Alert from "@mui/material/Alert";
+import InfiniteScroll from "react-infinite-scroll-component";
+import PropTypes from "prop-types";
+
+import MESSAGES from "config/constants/messages";
+import styles from "pages/styled";
+
+import NoteShared from "./NoteShared";
+
+const SharedList = ({ sharedNotes, displaySharedNote, pageShared, setPageShared }) => (
+  <div style={styles.sharedNotesGrid} id="scrollableShared" data-testid="scrollableShared">
+    {sharedNotes.length === 0 ? (
+      <Alert variant="filled" severity="info" id="saveNoteError" style={styles.noSharedNotes}>
+        {MESSAGES.NO_SHARED_NOTES}
+      </Alert>
+    ) : (
+      <InfiniteScroll
+        dataLength={sharedNotes.length}
+        next={() => setPageShared(pageShared + 1)}
+        hasMore={true}
```

Comment:
No need to write true directly. Just let option:
```suggestion
        hasMore
```